### PR TITLE
[#328] Multi insert

### PR DESCRIPTION
**Pull request by MattPerron**
Supports inserting multiple tuples in the same query. See [#328]

### CODE REVIEW

**Comment by MattPerron** [comment]
@malin1993ml Can you take a look at this?

**Comment by malin1993ml** [comment]
@MattPerron Sure.

File: src/planner/insert_plan.cpp [hunk 1]
```diff
@@ -61,95 +60,101 @@ InsertPlan::InsertPlan(parser::InsertStatement *parse_tree,
...
+          // If the column does not exist, insert a null value
+          if (pos >= query_columns->size()) {
+            //switch (elem.GetType()) {
+              //case common::Type::VARCHAR:
+              //case common::Type::VARBINARY: {
+              //  tuple->SetValue(col_cntr, *common::ValueFactory::GetNullStringValue(),
+              //                  GetPlanPool());
+              //  break;
+              //}
+              //default:
```
Comment (malin1993ml) [inline src/planner/insert_plan.cpp]: If these code are no longer useful, we can just delete them.

**Comment by MattPerron** [comment]
@malin1993ml It should already be rebased on top of master

File: src/planner/insert_plan.cpp (hunk 1 above)
Comment (MattPerron) [inline src/planner/insert_plan.cpp]: Sure, This is not code I added but it can be removed.

**Pull request merged**